hera_cal/redcal.py

```diff
@@ -1,5 +1,3 @@
-from hera_qm.datacontainer import DataContainer
```

Comment:
I think since we've officially decided that `hera_qm` is a dependency of `hera_cal`, this import statement can stay